### PR TITLE
Remove a bunch of unused methods in LoadBalancerProvider

### PR DESCRIPTION
**Pull request by ttomsu**
@cfieber @ajordens PTAL

@spinnaker/reviewers @gregturn @milismsft FYI

### CODE REVIEW

**Comment by ttomsu** [comment]
tag: https://github.com/spinnaker/spinnaker/issues/737

File: clouddriver-cf/src/main/groovy/com/netflix/spinnaker/clouddriver/cf/model/CloudFoundryLoadBalancerProvider.groovy [hunk 1]
```diff
@@ -29,34 +29,28 @@ class CloudFoundryLoadBalancerProvider implements LoadBalancerProvider<CloudFoun
   @Autowired
   CloudFoundryResourceRetriever cloudFoundryResourceRetriever
 
-  @Override
   Map<String, Set<CloudFoundryLoadBalancer>> getLoadBalancers() {
```
Comment (cfieber) [inline clouddriver-cf/src/main/groovy/com/netflix/spinnaker/clouddriver/cf/model/CloudFoundryLoadBalancerProvider.groovy]: Any objection to just removing the implementations (or @gregturn plan to do that as a followup? )

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleLoadBalancerProvider.groovy [hunk 2]
```diff
@@ -27,7 +27,6 @@ class GoogleLoadBalancerProvider implements LoadBalancerProvider<GoogleLoadBalan
   @Autowired
   GoogleResourceRetriever googleResourceRetriever
 
-  @Override
   Map<String, Set<GoogleLoadBalancer>> getLoadBalancers() {
```
Comment (cfieber) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleLoadBalancerProvider.groovy]: same here, may as well nuke the impls too

File: oort/oort-aws/src/main/groovy/com/netflix/spinnaker/oort/aws/provider/view/CatsLoadBalancerProvider.groovy [hunk 3]
```diff
@@ -49,7 +49,6 @@ class CatsLoadBalancerProvider implements LoadBalancerProvider<AmazonLoadBalance
     this.awsProvider = awsProvider
   }
 
-  @Override
   Map<String, Set<AmazonLoadBalancer>> getLoadBalancers() {
```
Comment (cfieber) [inline oort/oort-aws/src/main/groovy/com/netflix/spinnaker/oort/aws/provider/view/CatsLoadBalancerProvider.groovy]: and here - if you'd prefer each provider do the actual cleanup that is fine, but I'm ok with you killing off the impl in this class

File: clouddriver-cf/src/main/groovy/com/netflix/spinnaker/clouddriver/cf/model/CloudFoundryLoadBalancerProvider.groovy (hunk 1 above)
Comment (ttomsu) [inline clouddriver-cf/src/main/groovy/com/netflix/spinnaker/clouddriver/cf/model/CloudFoundryLoadBalancerProvider.groovy]: Done.

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleLoadBalancerProvider.groovy (hunk 2 above)
Comment (ttomsu) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleLoadBalancerProvider.groovy]: Done.

File: oort/oort-aws/src/main/groovy/com/netflix/spinnaker/oort/aws/provider/view/CatsLoadBalancerProvider.groovy (hunk 3 above)
Comment (ttomsu) [inline oort/oort-aws/src/main/groovy/com/netflix/spinnaker/oort/aws/provider/view/CatsLoadBalancerProvider.groovy]: Done.

**Pull request merged**